test_unstructured/documents/test_xml.py

```diff
@@ -44,6 +44,20 @@ def test_from_string(sample_document):
     assert type_tag.text.strip() == "10-K"
 
 
+def test_from_string_with_pre_tag():
+    sample_document = """
+    <pre>
```

Comment:
I think the original bug reported for this had HTML in the `<pre>` tag, like this:

```html
<pre>
<p> Hello there! </p>
</pre>
```

@christinestraub already added a test for that in #642, so I don't think there's anything additional we need testing wise there (unless you noticed/fixed an issue with it).

Reply:
Ah, thanks! Didn't see that since the pre tag tests were in the auto and html test files. I think this test is still needed, though, because xml makes a call to that part of code--I forgot the exact reason why, but I was getting a test failure (maybe with the html test?), so I was ensuring things worked for xml partition as well. I can remove it if you don't think it's needed.

Reply:
Yeah this is still a nice test to have too since the other one tests it from the partition function and not from the `XMLDocument` class. Thanks for adding it!